<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="recommendButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replyButton">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>댓글</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="boardContents">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>150</y>
     <width>671</width>
     <height>341</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim';&quot;&gt;Board Contents&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleRowLabel">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>90</y>
     <width>64</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="categoryRowLabel">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>90</y>
     <width>64</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>카테고리</string>
   </property>
  </widget>
  <widget class="QLabel" name="boardTitle">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>80</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Board Title</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="boardCategory">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>자유</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="boardWriter">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>user1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="writerRowLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>64</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>작성자</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gobackButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
